data/conditions.ts

```diff
@@ -715,7 +715,7 @@ export const Conditions: {[k: string]: ConditionData} = {
 		onEffectivenessPriority: -1,
 		onEffectiveness(typeMod, target, type, move) {
 			if (move && move.effectType === 'Move' && move.category !== 'Status' && type === 'Flying' && typeMod > 0) {
-				this.add('-activate', '', 'deltastream');
+				this.add('-activate', target, 'deltastream');
```

Comment:
```suggestion
				this.add('-fieldactivate', 'Delta Stream');
```
I think `-fieldactivate` is the most correct here since we're dealing with a weather and we don't need to send an empty string to not have a target.

Reply:
Nice! Presumably this would make it work similarly to Pay Day's " Coins were scattered everywhere!"?

Reply:
yep (even though Pay Day should probably use `-activate` since it's coming from a specific mon that everyone knows used the move)